content/en/docs/Upgrade/manual.md

```diff
@@ -69,4 +70,4 @@ You can also just generate that file yourself with the proper auth parameters li
 
 The auths map has an entry per registry, and the auth field contains your username and password encoded as HTTP 'Basic' Auth.
 
-NOTE: This means that your credentials are stored in plaintext. Have a look at the docker docs for the [credentials-store](https://docs.docker.com/engine/reference/commandline/login/#credentials-store)
\ No newline at end of file
+NOTE: This means that your credentials are stored in plaintext. Have a look at the docker docs for the [credentials-store](https://docs.docker.com/engine/reference/commandline/login/#credentials-store)
```

Comment:
worth moving into an alert now, but not a blocker

Reply:
weird, not sure why this appears as changed...I have not touched this..

Reply:
LOL let's ignore the messge then